lighthouse-core/report/html/renderer/crc-details-renderer.js

```diff
@@ -120,10 +121,10 @@ class CriticalRequestChainRenderer {
     }
 
     // Fill in url, host, and request size information.
-    const {file, hostname} = Util.parseURL(segment.node.request.url);
```

Comment:
tests are failing b/c of this change. `DetailsRenderer.renderTextURL` tosses the host part if `parsed.file === '/'`. Before, the chain renderer always used the results of `Util.parseURL` w/o post-processing. The chain renderer now looks like this when run on the root url of a site:

![image](https://user-images.githubusercontent.com/4071474/60039439-90695500-966b-11e9-8642-544c8df94431.png)


before:

![image](https://user-images.githubusercontent.com/4071474/60039474-a4ad5200-966b-11e9-9dbf-d13deff53a65.png)

Reply:
> The chain renderer now looks like this when run on the root url of a site:

this is an improvement, IMO.